src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -256,17 +249,13 @@ export const App = ({ displayToggle, toggleLoginModal }) => {
 
 App.propTypes = {
   toggleLoginModal: PropTypes.func.isRequired,
-  displayToggle: PropTypes.bool,
 };
-const mapStateToProps = state => ({
-  displayToggle: state?.featureToggles?.showDigitalForm1095b,
-});
 
 const mapDispatchToProps = dispatch => ({
   toggleLoginModal: open => dispatch(toggleLoginModalAction(open)),
 });
 
 export default connect(
-  mapStateToProps,
+  null,
```

Comment:
I initially removed this argument but found that it caused issues when clicking the "sign in" button. The issue seems to be due to the type requirements for the connect method, and it was easy to miss because of the nature of mocking and running the vets-website locally. Hopefully there aren't other gotchas, but this issue makes me less confident about this PR.

Reply:
Thanks for flagging this. I was able to verify at least one other place in the repo where the first argument is `null`. That file is `vets-website/src/applications/claims-status/containers/ClaimPage.jsx`, so I'm feeling relatively confident about this change.